### DISCUSSION TITLE
Obtaining sp_dc in windows.

### DISCUSSION THREAD

**Discussion by daedaevibin** [KRTirtho/spotube#1391] [Q&A]
### Is there an existing issue for this?

- [ ] I have searched the existing issues

### Current Behavior

It describes a super generic method which isn't available in Microsoft edge as you can not obtain this cookie from accounts.spotify.com and have to obtain it from open.spotify.com which isn't possible because you have to use an extension to right click. The "application" tab doesn't exist and there is no specific place to find cookie values at all.

### Expected Behavior

Easily obtain the value of sp_dc

### Steps to reproduce

Opened the app after install, followed provided instructions, and got nothing because the applications tab doesn't exist.

### Operating System

Windows 11

### Spotube version

Latest

### Installation source

Website (spotube.netlify.app) or (spotube.krtirtho.dev)

### Additional information

I'd like this information to be checked and updated properly so I don't have to resort to installing another web browser to obtain the value of sp_dc.

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by KRTirtho** [comment]
It's there. If you don't see the `Application` tab, you'll see a `+` icon or `>>` icon. Tapping it will reveal more available devtools tabs.

Or, if it's all too confusing, you can use [Cookie Editor](https://chromewebstore.google.com/detail/cookie-editor/hlkenndednhfkekhgcdicdfddnkalmdm) Chrome/Edge extension. Then, you can open `accounts.spotify.com` and copy the sp_dc cookie

**Comment by daedaevibin** [comment]
> It's there. If you don't see the `Application` tab, you'll see a `+` icon or `>>` icon. Tapping it will reveal more available devtools tabs.
> 
> Or, if it's all too confusing, you can use [Cookie Editor](https://chromewebstore.google.com/detail/cookie-editor/hlkenndednhfkekhgcdicdfddnkalmdm) Chrome/Edge extension. Then, you can open `accounts.spotify.com` and copy the sp_dc cookie



Ah, got it. That isn't mentioned and I really ever need to do something involved with dev tools so I wouldn't have known. Thanks for saying that, very helpful.
